### PR TITLE
[Template] Show the commit reference on the github comment

### PR DESCRIPTION
**Pull request by orta**
Handles the discussion in https://github.com/danger/danger-js/pull/97 ( this will then get merged into that PR )

### CODE REVIEW

File: source/runner/templates/github-issue-template.ts [hunk 1]
```diff
@@ -33,16 +33,32 @@ function table(name: string, emoji: string, violations: Array<Violation>): strin
...
+
+/**
+ * A commit aware signature that shows made by Danger or whatever
+ * @param {DangerResults} results Data to work with
+ * @returns {string} the signature in markdown
+ */
+export function signature(pr: GitHubPRDSL): string {
+  const url = '<a href="http://github.com/danger/danger-js/">dangerJS</a>'
+  const isFork = pr.head.user.login !== pr.base.user.login
+  if (isFork) {
```
Comment (kwonoj) [inline source/runner/templates/github-issue-template.ts:58]: maybe reduce duplication like below?

```js
const link = isFork? `[${pr.head.sha}](${pr.commits_url}/${pr.head.sha})` : pr.head.sha
return `Generated by :no_entry_sign:${url} against ${link}`
```

File: source/runner/Executor.ts [hunk 2]
```diff
@@ -40,10 +40,10 @@ export class Executor {
    * @param {string} file the filepath to the Dangerfile
    * @returns {void} It's a promise, so a void promise
    */
-
   async runDanger(file: string, runtime: DangerfileRuntimeEnv) {
     const results = await runDangerfileEnvironment(file, runtime)
-    await this.handleResults(results)
+    const signature = githubSignature(runtime.context.danger.github.pr)
```
Comment (kwonoj) [inline source/runner/Executor.ts:45]: executionwise it doesn't make any changes, but feeling building signature also need to be handled in `handleResults`?

Comment (macklinu) [reply source/runner/Executor.ts:45]: Passing the `runtime` (or at least `runtime.context`) into `handleResults()` could be beneficial for danger/danger-js#99 too (see https://github.com/danger/danger-js/pull/99#discussion_r98305986).

**Comment by macklinu** [comment]
Anything I can help with on this PR?

**Comment by kwonoj** [comment]
cleaning up old things and rebasing my fork to latest.

**Pull request closed without merging**